.gitignore

```diff
@@ -1,122 +0,0 @@
-Makefile
```

Comment:
Yes, I had some confusion with this file, because I have just started using git recently. I thought it was something accidentally added from my own file system and it prevented me from merging the master into my fork, so I ended up removing it. My intention was to not change this file in any way, so please ignore this change.